storage/src/main/resources/message/RemotePartitionDleteMetadataSnapshot.json

```diff
@@ -0,0 +1,42 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
There is a typo in this file name. Dlete -> Delete.
Does it make sense to correct it or leave the typo as it is?